code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -303,3 +303,64 @@
 /turf/closed/wall/mineral/plastitanium/copyTurf(turf/T)
 	. = ..()
 	T.transform = transform
+
+/////////////////////Pirate Plastitanium walls/////////////////////
+
+/turf/closed/wall/mineral/plastitaniumpirate
```

Comment:
Make this a subtype of `/turf/closed/wall/r_wall/syndicate`, just for the description.